apps/crn-server/test/data-providers/working-groups.data-provider.test.ts

```diff
@@ -271,5 +277,54 @@ describe('Working Group Data Provider', () => {
         expect(response?.leaders).toStrictEqual([]);
       });
     });
+
+    describe('Avatar', () => {
+      test.each`
+        user         | avatarText    | avatarValue              | expectedValue
+        ${`members`} | ${`null`}     | ${null}                  | ${undefined}
+        ${`members`} | ${`not null`} | ${[{ id: 'avatar-id' }]} | ${`${baseUrl}/api/assets/${appName}/avatar-id`}
+        ${`leaders`} | ${`null`}     | ${null}                  | ${undefined}
+        ${`leaders`} | ${`not null`} | ${[{ id: 'avatar-id' }]} | ${`${baseUrl}/api/assets/${appName}/avatar-id`}
+      `(
+        'Should parse the $user avatar correctly when the avatar is $avatarText',
+        async ({
+          user,
+          avatarValue,
+          expectedValue,
+        }: {
+          user: 'members' | 'leaders';
+          avatarValue: { id: string }[] | null;
+          expectedValue: string | undefined;
+        }) => {
+          const squidexGraphqlResponse =
+            getSquidexWorkingGroupGraphqlResponse();
+          const graphqlUser1 = getGraphQLUser();
+          graphqlUser1.flatData!.avatar = avatarValue;
+          if (user === 'leaders') {
+            squidexGraphqlResponse.findWorkingGroupsContent!.flatData.leaders =
+              [
+                {
+                  user: [graphqlUser1],
+                  role: 'Project Manager',
+                  workstreamRole: 'PM',
+                },
+              ];
+          } else {
+            squidexGraphqlResponse.findWorkingGroupsContent!.flatData.members =
+              [{ user: [graphqlUser1] }];
+          }
+
+          squidexGraphqlClientMock.request.mockResolvedValueOnce(
+            squidexGraphqlResponse,
+          );
+
+          const response = await workingGroupDataProvider.fetchById(
+            workingGroupId,
+          );
+
+          expect(response?.[user][0]?.user?.avatarUrl).toEqual(expectedValue);
```

Comment:
im not a huge fan of `?` in tests, we should know what the response is and i would prefer a `!` non-null assertion instead (which is allowed by linter in BE tests). Especially here you are expecting the property to be undefined and with the `?` you don't know which of the properties in the chain is undefined

Reply:
committed here https://github.com/yldio/asap-hub/pull/2359/commits/4b32e5e41fabf21c943b261f1d20eeab66f6a53b